manifests/b/Bluesound/BluOSController/3.18.1.1675/Bluesound.BluOSController.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Bluesound.BluOSController
+PackageVersion: 3.18.1.1675

```

```suggestion
PackageVersion: 3.18.1
```
The PackageVersion must match control panel; When updating this, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd>